guide/src/contributing/testing.md

```diff
@@ -34,8 +34,11 @@ cargo test -p webidl-tests --target wasm32-unknown-unknown
 These tests assert that we have reasonable error messages that point to the
 right source spans when the `#[wasm_bindgen]` proc-macro is misused.
 
+You can run these tests by running `cargo test` in the `crates/macro` directory:
+
 ```
-cargo test -p ui-tests
+cd crates/macro
+cargo test
```

Comment:
You can do this more concisely with `cargo test -p wasm-bindgen-macro`.

Reply:
ah yes, i missed that. was only looking at the members in the workspace, i'll push a commit shortly